### PR TITLE
add dubbogo tracing sample

### PR DESCRIPTION
**Pull request by luky116**

### CODE REVIEW

File: tracing/dubbo/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# RPC Dubbo for Dubbo-go 3.0
+
+For api definition and go client and server startup, please refer to [dubbo-go 3.0 quickstart](https://dubbogo.github.io/zh-cn/docs/user/quickstart/3.0/quickstart.html)
```
Comment (AlexStocks) [inline tracing/dubbo/README.md:3]: “For api definition and go client and server startup” --> If u wanna know how to startup the client/server and the definition of APIs,

File: tracing/dubbo/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Instructions
+1. Start `zookeeper` service in `docker/docker-compose.yml` file
+2. Start the server side.The server startup methods of golang and java are as follows:
```
Comment (AlexStocks) [inline tracing/dubbo/README.md:7]: 句号后面加上空格

File: tracing/dubbo/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+   1. Find the go-server folder, run the `main` function under the cmd package, and start the golang server
+   2. Find the java-server folder and execute `sh run.sh` to start the java server
+
+3. Start the client side.The client startup methods of golang and java are as follows:
```
Comment (AlexStocks) [inline tracing/dubbo/README.md:11]: 句号后面加上空格

File: tracing/dubbo/go-server/pkg/user.go [hunk 4]
```diff
@@ -19,49 +19,43 @@ package pkg
 
 import (
 	"context"
+	"fmt"
+	gxlog "github.com/dubbogo/gost/log"
```
Comment (AlexStocks) [inline tracing/dubbo/go-server/pkg/user.go:23]: reformat it

Comment (luky116) [reply tracing/dubbo/go-server/pkg/user.go:23]: 我reformat代码后，没啥变化，这个是哪里不合格式规范吗？

**Review by zouyx — COMMENTED** [review]
Why did you delete dubbo dir? had you verify this feature in dubbo?

**Review by AlexStocks — APPROVED** [review]

File: tracing/dubbo/go-client/cmd/client.go [hunk 5]
```diff
@@ -27,59 +27,44 @@ import (
...
-	_ "dubbo.apache.org/dubbo-go/v3/filter/filter_impl"
-	_ "dubbo.apache.org/dubbo-go/v3/protocol/dubbo"
-	_ "dubbo.apache.org/dubbo-go/v3/registry/protocol"
-	_ "dubbo.apache.org/dubbo-go/v3/registry/zookeeper"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
 
 	hessian "github.com/apache/dubbo-go-hessian2"
 
-	"github.com/dubbogo/gost/log"
+	"github.com/apache/dubbo-go-samples/tracing/dubbo/go-client/pkg"
```
Comment (cjphaha) [inline tracing/dubbo/go-client/cmd/client.go:36]: 这个放在第三个 import 块比较好

File: tracing/dubbo/go-server/cmd/server.go [hunk 6]
```diff
@@ -26,49 +26,41 @@ import (
...
 	"dubbo.apache.org/dubbo-go/v3/config"
-	_ "dubbo.apache.org/dubbo-go/v3/filter/filter_impl"
-	_ "dubbo.apache.org/dubbo-go/v3/protocol/dubbo"
-	_ "dubbo.apache.org/dubbo-go/v3/registry/protocol"
-	_ "dubbo.apache.org/dubbo-go/v3/registry/zookeeper"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
 
 	hessian "github.com/apache/dubbo-go-hessian2"
 
+	"github.com/apache/dubbo-go-samples/tracing/dubbo/go-server/pkg"
```
Comment (cjphaha) [inline tracing/dubbo/go-server/cmd/server.go:35]: 同上

File: tracing/dubbo/go-server/cmd/server.go [hunk 7]
```diff
@@ -26,49 +26,41 @@ import (
...
-// 		export APP_LOG_CONF_FILE="xxx"
 func main() {
-
+	config.SetProviderService(new(pkg.UserProvider))
 	hessian.RegisterPOJO(&pkg.User{})
-	config.Load()
-
-	initZipkin()
+	config.Load(config.WithPath("./tracing/dubbo/go-server/conf/dubbogo.yml"))
 	initSignal()
```
Comment (cjphaha) [inline tracing/dubbo/go-server/cmd/server.go:56]: 这里会阻塞住 initZipkin() 的运行吗？

**Comment by luky116** [comment]
> Why did you delete dubbo dir? had you verify this feature in dubbo?
我感觉tracing文件夹下面，只需要保留和tracing相关的代码就行了，java和go的互通，应该有单独的地方来测。这个和志信哥确认了下，可以删掉哈

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]

**Pull request merged**